static/js/protein/chart.ts

```diff
@@ -29,10 +29,61 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const tmpData = [
+    { name: "Cartilage", value: 3 },
+    { name: "Eye", value: 2 },
+    { name: "Liver", value: 1 },
+    { name: "Skin", value: 0 },
+  ];
+  const testData = data;
+  function convertData(val) {
+    if (val == "ProteinExpressionNotDetected") {
+      return 0;
+    } else if (val == "ProteinExpressionLow") {
+      return 1;
+    } else if (val == "ProteinExpressionMedium") {
+      return 2;
+    } else if (val == "ProteinExpressionHigh") {
+      return 3;
+    }
+  }
+  const reformat = data.map((item) => {
+    const newObj: any = {};
+    newObj["name"] = item.name;
+    newObj["value"] = convertData(item.value);
+    return newObj;
+  });
+
+  const size = reformat.map((x) => {
```

Comment:
nit: this looks like an array of names so maybe `names` instead of `size` for the variable name

Reply:
done !